docs/docs/winning-over-content-creators.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Winning Over Content Creators"
+issue: https://github.com/gatsbyjs/gatsby/issues/18281
+---
+
+Content creators are probably the heaviest end-users of your Gatsby projects because they create and manage the content used in your project. If you're a developer, you may find that simply talking to content creators about why _you_ like Gatsby is enough to get them interested. Below are topics a content creator cares about and how Gatsby can improve these.
```

Comment:
This isn't necessarily a true statement depending on the reader's situation, so we should keep it more open-ended.

```suggestion
Content creators can be potentially some of the heaviest end-users of your Gatsby projects when they create and manage the content used in your project. If you're a developer, you may find that simply talking to content creators about why _you_ like Gatsby is enough to get them interested. Below are topics a content creator cares about and how Gatsby can improve these areas.
```